CMakeLists.txt

```diff
@@ -643,6 +642,10 @@ macro(srt_make_application name)
 	set_target_properties(${name} PROPERTIES COMPILE_FLAGS "${CFLAGS_CXX_STANDARD} ${EXTRA_stransmit}" ${FORCE_RPATH})
 
 	target_link_libraries(${name} ${srt_link_library})
+
+	if (MICROSOFT)
+		target_link_libraries(${name} Iphlpapi)
```

Comment:
Won't this add iphlpapi.dll as dependency for *all* apps, not just "testing" which the only place where it is currently needed?

Reply:
This check is predicted to be done by all applications that make SRT connections, so this would be simply a complicated conditional slalom, to be withdrawn later anyway.

Reply:
This will be added to EVERY application, as long as explicitly requested. For Linux & Mac it will be always done (needs no extra features from the system). For Windows, it requires extra `ENABLE_CONSELF_CHECK_WIN32` flag as it implies linking against extra `Iphlpapi.lib`.